File: web3rpc/rpc-specs/api/code-samples/javascript/eth/transaction/getTransactionReceipt.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));

    const transactionHash = '0xc6acc62baaa57483da8d5e08aaed1907d82f0e25bd553ce3745ef1bc7b7f4476'

    sdk.eth.getTransactionReceipt(transactionHash, {}, (err, data, response) => {
        console.log(data);
    });

}
)()

File: web3rpc/rpc-specs/code-samples/python/eth/transaction/getTransactionReceipt.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

blockHash = "0xaca5d9a1ed8b86b1ef61431b2bedfc99a66eaefc3a7e1cffdf9ff53653956a67"

w3 = Web3(Web3.HTTPProvider(host))
eth_response = w3.eth.get_transaction_receipt(blockHash)

print(eth_response)
